src/modules/job/job.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, HttpCode, HttpStatus, Post } from '@nestjs/common';
+import { JobService } from './job.service';
+import { CreateJobDTO } from './dto/create-job.dto';
+import { ApiBearerAuth, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+
+@ApiBearerAuth()
+@ApiTags('Organisation')
+@Controller('jobs')
+export class JobController {
+  constructor(private readonly jobService: JobService) {}
+
+  @ApiOperation({ summary: 'Create Job Listing' })
+  @ApiResponse({
```

Comment:
the convention to follow is status_code not status

Reply:
I am not the one who created that. I only used it and tampering with that will cause a lot of conflicts

@davidolanrewaju